cli/status/status.go

```diff
@@ -20,35 +68,110 @@ type StatusOpts struct {
 func Status(runningCtx running.RunningCtx, opts StatusOpts) error {
 	ts := table.NewWriter()
 	ts.SetOutputMirror(os.Stdout)
-	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE"})
+	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE", "ALERTS"})
 
 	for _, run := range runningCtx.Instances {
 		row := []interface{}{}
 		fullInstanceName := running.GetAppInstanceName(run)
 		procStatus := running.Status(&run)
-
 		row = append(row, fullInstanceName)
-		row = append(row, procStatus.ColorSprint(procStatus.Status))
-		if procStatus.Code == process_utils.ProcessRunningCode {
-			row = append(row, procStatus.PID)
-		}
+		warnings := []string{}
+		mode_str := ""
```

Comment:
I understand it as follows. We need these variables to save results to add them later as columns in some specified order. But I don't see why not to produce the columns in the order how they should appear in the row.

I see that we use network requests only if the network connection succeeds, i.e. under the `if err == nil` branch. However, we can save some `isConnected` property and use it where appropriate. Maybe we can even use just `conn != nil` for it.

This way we can produce the needed values and add them into corresponding columns right after this. It should make the code easier to follow.